examples/posix_sockets/Makefile

```diff
@@ -16,6 +16,7 @@ BOARD_INSUFFICIENT_MEMORY := airfy-beacon chronos msb-430 msb-430h nrf51dongle n
 # NOTE: 6LoWPAN will be included if IEEE802.15.4 devices are present
 USEMODULE += gnrc_netdev_default
 USEMODULE += auto_init_gnrc_netif
+USEMODULE += gnrc_txtsnd
```

Comment:
True - I was a bit lazy and only used `git grep` without looking at the context. Fixed.